assets/src/js/page_header_parallax.js

```diff
@@ -0,0 +1,8 @@
+export const setupPageHeaderParallax = () => {
+  const pageHeaders = [...document.querySelectorAll('.is-pattern-p4-page-header > .wp-block-media-text__media > img')];
```

Comment:
I don't see necessary a spread syntax (...) here.
`const pageHeaders = document.querySelectorAll('.is-pattern-p4-page-header > .wp-block-media-text__media > img');` it's fine

Reply:
It works fine indeed, but it's not an array, it's a `NodeList` which is a bit different. It's ok for now, but it might cause issues in case we want to use some other array-specific functions 🙂 I can change it if you prefer 👍

Reply:
> One limitation of this implementation is that it will only work if the Page Header pattern is placed first in the page (also without a page title) but I guess we should take into account that it could be anywhere on the page? I don't think we enforce its positioning at the moment 🤔

Gotcha! Did you try if setting `background-attachment` as `fixed` to the image behaves similar?

Reply:
It's not gonna work here, because the Media & Text block doesn't use `background` CSS properties but an actual image tag... I think it should be doable however to find the position of the Page Header, and determine the scrolling effect based on that!